Kuroba/app/src/main/java/com/github/adamantcheese/chan/ui/layout/ThreadLayout.java

```diff
@@ -586,6 +602,14 @@ public void onRestoreRemovedPostsClicked(
         fixSnackbarText(getContext(), snackbar);
     }
 
+    @Override
+    public void shownBackgroundWatcherIsDisabledToast() {
```

Comment:
Maybe it is a better idea to make this a snackbar with a button that will bring user to the background watcher settings?

Reply:
Maybe later.